locale/en/locale.po

```diff
@@ -164,3 +164,33 @@ msgid "plugins.generic.crossref.successMessage"
 msgstr ""
 "The metadata for this item has been deposited with Crossref. "
 "To view further details, see the submission in the <a href=\"https://doi.crossref.org/servlet/submissionAdmin?sf=detail&submissionID={$submissionId}\">Crossref admin panel</a>."
+
+msgid "plugins.generic.crossref.issue.required"
+msgstr "{$pluginName} : Issue is required"
```

Comment:
Remove `{$pluginName}` from these in favour of just having the text fully specify the error message. `Crossref XML Export Plugin :` is long-winded, and this will be confusing to translators rather than self-contained messages like `The submission "{$publicationTitle}" is not associated with an issue and cannot be deposited with CrossRef.`